pkg/controller/infra/infra_controller.go

```diff
@@ -252,11 +629,20 @@ func (c *InfraController) syncCluster(key string) error {
 		return err
 	}
 
-	err = c.ansibleRunner.RunPlaybook(cluster.Namespace, cluster.Name, jobPrefix, infraPlaybook,
-		provisionInventoryTemplate, vars)
+	job, cfgMap := c.ansibleGenerator.GeneratePlaybookJob(cluster, jobPrefix, infraPlaybook, provisionInventoryTemplate, vars)
+
+	_, err = c.kubeClient.CoreV1().ConfigMaps(cluster.Namespace).Create(cfgMap)
 	if err != nil {
 		return err
 	}
 
-	return nil
+	_, err = c.kubeClient.BatchV1().Jobs(cluster.Namespace).Create(job)
```

Comment:
Are we punting on cleaning up the jobs and configmaps for the time being? When we get to that, we need to make sure that we are resilient to the scenario where the configmap is created successfully but the job fails--ensuring that the configmap is cleaned up in that case.

Reply:
for now, yes, we are -- we need to add it to the todo list